lib/Frontend/OutputFileMap.cpp

```diff
@@ -108,21 +100,42 @@ void OutputFileMap::dump(llvm::raw_ostream &os, bool Sort) const {
   }
 }
 
-bool OutputFileMap::parse(std::unique_ptr<llvm::MemoryBuffer> Buffer,
-                          StringRef workingDirectory) {
+void OutputFileMap::write(llvm::raw_ostream &os,
+                          ArrayRef<StringRef> inputs) const {
+  auto printQuotedString = [&os](StringRef s) -> void {
+    os << "\"" << s << "\"";
+  };
+  for (const auto input : inputs) {
+    const TypeToPathMap *outputMap = getOutputMapForInput(input);
+    if (!outputMap)
+      continue;
+    for (auto &typeAndOutputPath : *outputMap) {
+      printQuotedString(input);
+      os << " :\n";
```

Comment:
Nitpick: typical YAML style would not put a space here.

Reply:
Tx. Fixed.